lib/sampler.py

```diff
@@ -80,6 +82,16 @@ def set_queue(self, queue, worker_id, worker_name):
         self._queue = queue
         self._watcher = None
 
+    def rss_summary(self):
+        """Return dictionary of collected maximum RSS values per each
+           run task, set with task_id keys.
+        """
+
+        return self._rss_summary
+
+    def sample_interval(self):
+        return self._watcher._sample_interval
```

Comment:
The rule of thumb is to don't access external fields, which start from the underscore. I think we can apply the term 'external' either to code outside of the file or outside of the class. I would prefer the latter.

So, I would define `sample_interval` property in SamplerWather and would access the field using it.

BTW, it if we'll define `sample_interval` and `rss_summary` as properties, it would more in sync with the surrounding code.

Reply:
Ok, done.